main.js

```diff
@@ -1 +1,60 @@
 import customers from './customers.js'
+
+
+
+let person = [{
+  thumbNail: 'https://randomuser.me/api/portraits/thumb/women/89.jpg',
+  name: "Pauline Hughes",
+  email: "pauline.hughes@example.com" ,
+  address: "3747 Elgin St, Roseville, IN 93281"
+ 
+}]
+
+function listCustomers(personArray) {
+  let customerList = document.querySelector("#customer-list");
+  console.log(customerList);
+
+  for (let customer of customers) {
```

Comment:
Nice work so far. You'll want to loop over the parameter, `personArray`, rather than a specific array.